dtests/src/resources/scripts/cdcConnector/createAndLoadFewTables.sql

```diff
@@ -19,30 +19,62 @@ CREATE EXTERNAL TABLE staging_BANK
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BANK.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
 CREATE TABLE BANK USING column OPTIONS(partition_by 'BNK_ORG_ID', buckets '32',key_columns 'CLIENT_ID,BNK_ORG_ID,BNK_ID ' ) AS (SELECT * FROM staging_BANK);
 
+DROP TABLE IF EXISTS BENEFIT_PACKAGE;
+DROP TABLE IF EXISTS staging_BENEFIT_PACKAGE;
+CREATE EXTERNAL TABLE IF NOT EXISTS  staging_BENEFIT_PACKAGE
+    USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BENEFIT_PACKAGE.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
+CREATE TABLE IF NOT EXISTS BENEFIT_PACKAGE USING column OPTIONS(partition_by 'BENE_PKG_ID',redundancy '1', buckets '32',key_columns 'CLIENT_ID,BENE_PKG_ID' ) AS (SELECT * FROM staging_BENEFIT_PACKAGE);
+
 DROP TABLE IF EXISTS BANK_ACCOUNT;
 DROP TABLE IF EXISTS staging_BANK_ACCOUNT;
 ----- CREATE TEMPORARY STAGING TABLE TO LOAD CSV FORMATTED DATA -----
-CREATE EXTERNAL TABLE staging_BANK_ACCOUNT
+CREATE EXTERNAL TABLE IF NOT EXISTS  staging_BANK_ACCOUNT
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BANK_ACCOUNT.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
-CREATE TABLE BANK_ACCOUNT USING column OPTIONS(partition_by 'BNK_ORG_ID',buckets '32',key_columns 'CLIENT_ID,BNK_ORG_ID,BNK_ID,BNK_ACCT_ID ' ) AS (SELECT * FROM staging_BANK_ACCOUNT);
+ CREATE TABLE IF NOT EXISTS BANK_ACCOUNT USING column OPTIONS(partition_by 'BNK_ORG_ID',redundancy '1', buckets '32',key_columns 'CLIENT_ID,BNK_ORG_ID,BNK_ID,BNK_ACCT_ID' ) AS (SELECT * FROM staging_BANK_ACCOUNT);
 
 DROP TABLE IF EXISTS BENEFIT_GROUP_NAME;
 DROP TABLE IF EXISTS staging_BENEFIT_GROUP_NAME;
 ----- CREATE TEMPORARY STAGING TABLE TO LOAD CSV FORMATTED DATA -----
-CREATE EXTERNAL TABLE staging_BENEFIT_GROUP_NAME
+CREATE EXTERNAL TABLE IF NOT EXISTS  staging_BENEFIT_GROUP_NAME
     USING com.databricks.spark.csv OPTIONS (path '/export/shared/QA_DATA/TMG_Data_20G/BENEFIT_GROUP_NAME.dat', header 'true', inferSchema 'true', nullValue 'NULL', maxCharsPerColumn '4096');
```

Comment:
Parametrize the variables in sql script and pass them through bt or conf file at runtime. Please check for all the sql scripts.

Reply:
Done